src/components/shared/SampleTooltipContents.js

```diff
@@ -117,6 +121,22 @@ export class SampleTooltipContents extends React.PureComponent<Props> {
       categories,
       implementationFilter,
     } = this.props;
+
+    const { samples, stackTable } = rangeFilteredThread;
+    const stackIndex = samples.stack[sampleIndex];
+    const hasSamples = samples.length > 0 && stackTable.length > 1;
+    let hasStack = false;
+    if (hasSamples) {
+      const stack = getFuncNamesAndOriginsForPath(
+        convertStackToCallNodeAndCategoryPath(
+          rangeFilteredThread,
+          ensureExists(stackIndex)
+        ),
+        rangeFilteredThread
+      );
+      hasStack = stack.length > 1 || stack[0].funcName !== '(root)';
+    }
```

Comment:
It's a good idea to hide the stacks if we only have `(root)`! It wasn't looking good anyways.